config.xml

```diff
@@ -51,9 +51,12 @@
     <preference name="StatusBarOverlaysWebView" value="false" />
     <preference name="StatusBarBackgroundColor" value="#151A1E" />
     <preference name="BackgroundColor" value="0xFF151A1E" />
-    <preference name="xwalkVersion" value="22" />
+    <preference name="xwalkVersion" value="20" />
```

Comment:
I guess it has been automatically added. Please remove all crosswalk related changes here since I already made changes